baseapp/baseapp.go

```diff
@@ -2,6 +2,7 @@ package baseapp
 
 import (
 	"context"
+	goerrors "errors"
```

Comment:
why do we have 3 different error packages being used here?

Reply:
golang errors was added to get the Join() method recommended here:
https://github.com/cosmos/cosmos-sdk/pull/17094#discussion_r1286754362

the other error packages were selected prior to this PR

is there another function/package you prefer to use?

Reply:
https://github.com/cosmos/cosmos-sdk/pull/17094#discussion_r1305631590